### ISSUE TITLE
Number of tests?

### ISSUE THREAD

**Issue by dejan-shuki** [CSSEGISandData/COVID-19#2177]
These numbers are misleading without some supporting data like - number of tests performed.

**Comment by MelbourneDeveloper** [comment]
@dejan-shuki I agree to some extent, but it's hard enough to get this data right. If there is more data added, it will just end up consuming more time. I think the best way is to pull test data from an external, well - verified source and aggregate it with this data through a separate process.
